src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -170,7 +170,7 @@ module.exports = function<T, P, I, TI, PI, C, CX>(config : HostConfig<T, P, I, T
 
     deferredUpdates,
 
-    getPublicRootInstance(container : OpaqueNode) : (ReactComponent<any, any, any> | I | TI | null) {
+    getPublicRootInstance(container : OpaqueNode) : (ReactComponent<any, any, any> | PI | null) {
```

Comment:
Since this exists for Stack compat I would return nothing. Stack doesn't allow rendering top-level text nodes so nothing would break if we didn't implement this.

If we *did* want to implement it (I don't see why) then it would make sense to return the TextInstance (text node), similar to how `ReactDOM.render(<div />)` returns the Instance (div).